Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -0,0 +1,222 @@
+        didSet {
+            if oldValue != channel {
+                updateContent()
+            }
+        }
+    }
+    
+    // MARK: - Subviews
+    
+    public private(set) lazy var container: ContainerStackView = {
+        let stack = ContainerStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        return stack
+    }()
+    
+    public private(set) lazy var avatarView: ChatChannelAvatarView<ExtraData> = {
+        let avatar = uiConfig.channelList.channelListItemUI.avatarView.init()
+        avatar.translatesAutoresizingMaskIntoConstraints = false
+        return avatar
+    }()
+    
+    public private(set) lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    public private(set) lazy var subtitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    public private(set) lazy var unreadCountBubble: UnreadCountBubbleView = {
+        let bubble = uiConfig.channelList.channelListItemUI.unreadCountView.init()
+        bubble.translatesAutoresizingMaskIntoConstraints = false
+        return bubble
+    }()
+    
+    public private(set) lazy var readStatusView: ReadStatusCheckmarkView = {
+        let view = uiConfig.channelList.channelListItemUI.readStatusView.init()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    public private(set) lazy var timestampLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    // MARK: - Init
+    
+    public required init(
+        channel: _ChatChannel<ExtraData>? = nil,
+        uiConfig: UIConfig<ExtraData> = .default
+    ) {
+        self.channel = channel
+        self.uiConfig = uiConfig
+        
+        super.init(frame: .zero)
+        
+        commonInit()
+    }
+    
+    public required init?(coder: NSCoder) {
+        self.uiConfig = .default
+        self.channel = nil
+        
+        super.init(coder: coder)
+        
+        commonInit()
+    }
+    
+    public func commonInit() {
+        setupAppearance()
+        setupLayout()
+        updateContent()
+    }
+    
+    // MARK: - Public
+    
+    open func setupAppearance() {
+        titleLabel.numberOfLines = 1
+        titleLabel.font = .boldSystemFont(ofSize: .scaledFontSize(for: 14))
+        
+        subtitleLabel.numberOfLines = 1
+        subtitleLabel.textColor = .gray
+        subtitleLabel.font = .systemFont(ofSize: .scaledFontSize(for: 12.5))
+        
+        timestampLabel.numberOfLines = 1
+        timestampLabel.textColor = .gray
+        timestampLabel.font = .systemFont(ofSize: .scaledFontSize(for: 12),
+                                          weight: .medium)
+    }
```

Comment:
I'd also make labels adjusting the font size when content size category changes:
```swift 
label.adjustsFontForContentSizeCategory = true
```